soc/silabs/silabs_siwx91x/CMakeLists.txt

```diff
@@ -12,3 +17,36 @@ set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
     ${KERNEL_BIN_NAME}
     ${KERNEL_BIN_NAME}.rps
 )
+
+# Custom signing with Silicon Labs Commander
+if(CONFIG_SiWX91X_SILABS_COMMANDER_SIGN)
+    # Define input and output files
+    set(SIGNED_RPS ${PROJECT_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME}.bin.rps)
```

Comment:
Do you need absolute path? I believe you need it only because you change `WORKING_DIRECTORY` below. You can probably remove remove all this complexity.